content/gocourse-2019-05-melbourne.slide

```diff
@@ -1003,6 +1003,46 @@ Use `gofmt` or `goimport` as part of your editor/IDE setup
 - ...
 
 
+* JSON - Marshal / Unmarshal
```

Comment:
I'd calling  `(De-)Serialisation` cause in golang:
Marshal => String
Encode => Stream